pymeasure/instruments/hp/hp856Xx.py

```diff
@@ -1392,131 +1609,1374 @@ def marker_minimum(self):
+            (72, 16, 712, 766)
+
+        """,
+        maxsplit=4,
+        cast=int,
+        get_process=tuple
+    )
+
+    def plot(self, p1x, p1y, p2x, p2y):
+        """Copies the specified display contents onto any HP-GL plotter. Set
+        the plotter address to 5, select the Pi and P2 positions, and then
+        execute the plot command. P1 and P2 correspond to the lower-left and
+        upper-right plotter positions, respectively. If P1 and P2 are not
+        specified, default values (either preloaded from power-up or sent in
+        via a previous plot command) are used. Once PLOT is executed, no
+        subsequent commands are executed until PLOT is done.
+
+        :param p1x: plotter-dependent value that specify the lower-left plotter position x-axis
+        :type p1x: int
+        :param p1y: plotter-dependent value that specify the lower-left plotter position y-axis
+        :type p1y: int
+        :param p2x: plotter-dependent values that specify the upper-right plotter position x-axis
+        :type p2x: int
+        :param p2y: plotter-dependent values that specify the upper-right plotter position y-axis
+        :type p2y: int
+        """
+        if not (isinstance(p1x, int) or isinstance(p1y, int) or isinstance(p2x, int) or
+                isinstance(p2y, int)):
+            raise TypeError("Should be of type int")
+
+        # TODO: Value check
+
+        self.write("PLOT %d,%d,%d,%d" % (p1x, p1y, p2x, p2y))
+
+    @property
+    def display_origins(self):
+        raise NotImplementedError()
+
+    @property
+    def plot_source(self):
+        raise NotImplementedError()
+
+    def print(self, active):
+        raise NotImplementedError()
+
+    protect_state = Instrument.control(
+        "PSTATE?", "PSTATE %s",
+        """
+        Control the storing of any new data in the state or trace registers.
+        If set to 'True', the registers are “locked”; the data in them cannot be erased or
+        overwritten, although the data can be recalled. To “unlock” the registers, and store new
+        data, set 'protect_state' to off by selecting 'False' as the parameter.
+
+        Type: :code:`bool`
+        """,
+        map_values=True,
+        validator=strict_discrete_set,
+        values={True: "1", False: "0"},
+        cast=str
+    )
+
+    def power_bandwidth(self, trace, percent):
+        """Measure the combined power of all signal responses contained in a
+        trace array. The command then computes the bandwidth equal to a
+        percentage of the total power. For example, if 100% is specified, the
+        power bandwidth equals the current frequency span. If 50% is specified,
+        trace elements are eliminated from either end of the array, until the
+        combined power of the remaining trace elements equals half of the total
+        power computed. The frequency span of these remaining trace elements is
+        the power bandwidth output to the controller.
```

Comment:
It is a method, so please formulate it as an action:
`get_power_bandwidth`.

You could implement the traces with the [Channels class](https://pymeasure.readthedocs.io/en/latest/dev/adding_instruments/channels.html), if you want to.